src/ml_modules/bert_classifier.py

```diff
@@ -29,13 +23,22 @@ class BertClassifier(object):
     ["KURTARMA"]
     """
 
-    def __init__(self, classification_threshold=CLASSIFICATION_THRESHOLD):
+    def __init__(self, classification_threshold=0.5):
+        # The score threshold to deem a label as positive.
         self.classification_threshold = classification_threshold
 
-    def __query(self, text):
+        API_TOKEN = os.getenv("HF_HUB_TOKEN")
```

Comment:
I think these should remain as file level constants (maybe using dunder to indicate privateness). Both for better readability, but also they shouldn't be exposed outside with the class import (since they are intended to be hardcoded)

Reply:
Moved API key token and model name to static-definition area.